gateway/platforms/vertx3/vertx3/src/main/java/io/apiman/gateway/platforms/vertx3/engine/VertxPluginRegistry.java

```diff
@@ -185,7 +185,7 @@ private HttpClientOptions configureHttpClientOptions(final URL artifactUrl) {
             if (!"none".equalsIgnoreCase(noProxy)) noProxyAddresses.add(InetAddress.getByName(noProxy));
         } catch (UnknownHostException e) {
             //Non blocking error while trying to get no proxy addresses. Proxy will be always applied if configured.
-            e.printStackTrace();
+            //e.printStackTrace();
```

Comment:
I have a separate ticket for this. The whole class is a bit of a mess and needs to be reworked. Check out https://github.com/apiman/apiman/issues/971 - (this was just to avoid obscuring what I was doing at the time and was not going to be in the non-draft commit)